.github/actions/cdn_deployment_aws/action.yaml

```diff
@@ -0,0 +1,47 @@
+name: CDN Deployment for AWS
+description: Deploys to AWS CDN and optionally invalidates the path in CloudFront
+inputs:
+  environment:
+    description: Environment to deploy to
+    default: 'Integ'
+  artifact:
+    description: Name of the artifact
+    required: true

```

Comment:
Does this need `type: boolean` ? Same with `aws_distribution_id` and `aws_bucket_name`

Reply:
You can't set type on action inputs - who knows why